packages/server/graphql/public/mutations/acceptRequestToJoinDomain.ts

```diff
@@ -31,7 +32,7 @@ const acceptRequestToJoinDomain: MutationResolvers['acceptRequestToJoinDomain']
   const request = await pg
     .selectFrom('DomainJoinRequest')
     .selectAll()
-    .where('id', '=', requestId)
+    .where('id', '=', fromGlobalId(requestId).id)
```

Comment:
+1 I like the `fromGlobalId` and `toGlobalId` functions. I still think we should create a `gqlIds/DomainJoinRequestId` to wrap these and call `fromGlobalId` and `toGlobalId` internally. This has the advantage to be consistent with the other places and probably easier to find when figuring out how to decode the id.

Reply:
Added